orm/encoding/ormfield/codec_test.go

```diff
@@ -99,7 +97,7 @@ func TestCompactUInt32(t *testing.T) {
 	testEncodeDecode(1073741823, 4)
 	testEncodeDecode(1073741824, 5)
 
-	// randomized tests
+	// Randomized tests.
```

Comment:
Curious, is it a linter doing this? If so, which one? I think this has a too big impact to run such linter in the whole SDK.

Reply:
I believe I have disabled it.  Also I agree. Let me poke about on that.